tap_zendesk/http.py

```diff
@@ -22,12 +22,14 @@ def is_fatal(exception):
                       requests.exceptions.HTTPError,
                       max_tries=10,
                       giveup=is_fatal)
-def call_api(url, params, headers):
-    response = requests.get(url, params=params, headers=headers)
+@backoff.on_exception(backoff.expo,Timeout, #As timeout error does not have attribute status_code, hence giveup does not work in this case.
```

Comment:
We are adding max_tries=5 with factor=2 so please update accordingly to give significant delay between 2 retries.

Reply:
Updated it.